tests/sqla_models_tests.py

```diff
@@ -223,6 +223,28 @@ def test_incorrect_jinja_syntax_raises_correct_exception(self):
             with pytest.raises(QueryObjectValidationError):
                 table.get_sqla_query(**query_obj)
 
+    def test_query_format(self): 
```

Comment:
```suggestion
    def test_query_format_strip_trailing_semicolon(self): 
```

Let's be more specific here